pkg/cmd/server/origin/authorizer.go

```diff
@@ -56,15 +57,18 @@ func newAuthorizer(
 	subjectLocator := authorizer.NewSubjectLocator(kubeSubjectLocator)
 
 	scopeLimitedAuthorizer := scope.NewAuthorizer(roleBasedAuthorizer, clusterRoleGetter, messageMaker)
+	// Wrap with an authorizer that detects unsafe requests and modifies verbs/resources appropriately so policy can address them separately
+	browserSafeAuthorizer := browsersafe.NewBrowserSafeAuthorizer(scopeLimitedAuthorizer, bootstrappolicy.AuthenticatedGroup)
 
 	graph := node.NewGraph()
 	node.AddGraphEventHandlers(graph, podInformer, pvInformer)
 	nodeAuthorizer := node.NewAuthorizer(graph, nodeidentifier.NewDefaultNodeIdentifier(), kbootstrappolicy.NodeRules())
 
 	authorizer := authorizerunion.New(
-		authorizerfactory.NewPrivilegedGroups(user.SystemPrivilegedGroup), // authorizes system:masters to do anything, just like upstream
+		authorizerfactory.NewPrivilegedGroups(bootstrappolicy.MastersGroup), // authorizes system:masters to do anything, just like upstream
```

Comment:
Why not use the kube constant?

Reply:
Allowed me to drop an import.

Reply:
I'd rather stick with the upstream constant